### PR TITLE
init url-generator lib

### PR DESCRIPTION
**Pull request by arnaudruffin**
The lib we discussed is ready.
In this branch the lib is initialized when the app is started but not used in the urls yet. I don't know if you would like to merge it as it is or wait for more (I won't have time for more right now...)

### CODE REVIEW

**Comment by antechrestos** [comment]
I don't see what is exported from your library

**Comment by arnaudruffin** [comment]
The library is loaded by the ioserver.js.
It will expose the `generateUrl` method to the rest of the code

**Comment by arnaudruffin** [comment]
It should be ready now

File: routes/receive.js [hunk 1]
```diff
@@ -35,16 +35,16 @@ debug.log = console.log.bind(console);
...
+    dao.getSenderFromUri(uri, function (sender) {
+        debug('receive - rendering receive for uri %s', uri);
         res.render('receive', {
             title: "Just drop it",
             isLocal: typeof process.env.OPENSHIFT_NODEJS_IP === "undefined",
             fileName: sender.fileName,
             fileSize: sender.fileSize,
             jdropitVersion: global.DROP_IT_VERSION,
-            senderId: fileId
+            senderId: sender.senderId
```
Comment (antechrestos) [inline routes/receive.js]: I thinck you must keep sender id cause server rooting is based on it, otherwise message may be lost

Comment (arnaudruffin) [inline routes/receive.js]: do you mean "routing"? because it this case no, the routing is now based on the fileId

**Pull request merged**